packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue

```diff
@@ -42,13 +42,16 @@
 		</div>
 		<div :tabName="StratifyTabs.Notebook">
 			<tera-drilldown-section class="notebook-section">
-				<tera-notebook-jupyter-input
-					:kernel-manager="kernelManager"
-					:default-options="[]"
-					:context-language="'python3'"
-					@llm-output="(data: any) => processLLMOutput(data)"
-				/>
-
+				<div class="toolbar">
+					<tera-notebook-jupyter-input
+						:kernel-manager="kernelManager"
+						:default-options="[]"
+						:context-language="'python3'"
+						@llm-output="(data: any) => processLLMOutput(data)"
+						@llm-thought-output="(data: any) => (llmThought = data)"
+					/>
+					<tera-notebook-jupyter-thought-output :llmThought="llmThought" />
+				</div>
```

Comment:
This pattern with the toolbar div and the components inside is repetitive. The left padding should just be default spacing. We should save this for later - just wanted to point it out.